provisioner/secret.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var base = require('taskcluster-base');
+var assert = require('assert');
+var debug = require('debug')('aws-provisioner:Secret')
+var taskcluster = require('taskcluster-client');
+
+var Secret = base.Entity.configure({
+  version: 1,
+
+  // TODO: verify these are valid azure keys - heeeelp please Jonas
+  partitionKey: base.Entity.keys.StringKey('token'),
+  rowKey: base.Entity.keys.StringKey('provisionerId'),
```

Comment:
yep, originally you had suggested that we use one table with a provisioner id field, but this makes much more sense to me :)